benefit/data/sheets/WorkBasketResultFields/WorkBasketResultFields.json

```diff
@@ -1,41 +1,41 @@
 [
   {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "CaseFieldID": "[CASE_REFERENCE]", "Label": "Case Ref", "DisplayOrder": 1},
   {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "CaseFieldID": "appeal", "ListElementCode": "appellant.name.lastName", "Label": "Surname", "DisplayOrder": 2},
```

Comment:
Looking at git history, this has been implemented since 2020, I'm reticent to change it and add in an AccessProfile if it's working as intended without a UserRole. In theory, we could add a base/default profile such as a caseworker or caseworker-sscs (as that covers all roles sans citizen I believe in the RoleToAccessProfiles.json mapping), but will leave it for now.